parser_library/fuzzer.dict

```diff
@@ -0,0 +1,41 @@
+"&VAR_SYM"
+".SEQ_SYM"
+"ORD_SYM"
+"SETA"
+"SETB"
+"SETC"
+"LCLA"
+"LCLB"
+"LCLC"
+"GBLA"
+"GBLB"
+"GBLC"
+"ANOP"
+"ACTR"
+"AGO"
+"AIF"
+"MACRO"
+"MEND"
+"MEXIT"
+"ASPACE"
+"AEJECT"
+"CSECT"
+"DSECT"
+"RSECT"
+"COMM"
+"LOCTR"
+"EQU"
+"DC"
+"DS"
+"COPY"
+"EXTRN"
+"ORG"
+"L'"
+"I'"
+"S'"
+"T'"
+"D'"
+"O'"
+"N'"
+"K'"
+"C'ABC'"
```

Comment:
Have you thought about adding some machine instructions as well?

Reply:
Yes, I will add more extensive dictionary in future commits.